pulpanalytics/management/commands/summarize.py

```diff
@@ -43,16 +43,22 @@ def _handle_online_workers(systems, summary):
         online_workers_qs = OnlineWorkers.objects.filter(system__in=systems)
         online_workers_stats = online_workers_qs.aggregate(Avg('processes'), Avg('hosts'))
 
-        summary.online_workers.processes__avg = online_workers_stats['processes__avg']
-        summary.online_workers.hosts__avg = online_workers_stats['hosts__avg']
+        try:
+            summary.online_workers.processes__avg = online_workers_stats['processes__avg']
+            summary.online_workers.hosts__avg = online_workers_stats['hosts__avg']
+        except TypeError:
+            pass
```

Comment:
Is there a way we can avoid adding these? It's in the spec so we should expect it's their and if not have it raise an exception.

Reply:
I see here that this is actually optional in the protobuf so +1 to this. Thanks!